src/app/collective-rewards/leaderboard/BuilderLeaderBoard.tsx

```diff
@@ -20,9 +24,24 @@ export const BuildersLeaderBoard = () => {
         <CollapsibleTrigger>
           <div className="flex items-center justify-between w-full">
             <HeaderTitle className="">Rewards leaderboard</HeaderTitle>
-            <Button variant="primary" onClick={onManageAllocations} disabled={canManageAllocations}>
-              Manage Allocations
-            </Button>
+
+            <Popover
+              content={
+                <>
+                  <p>Distribution in progress.</p>
+                </>
+              }
+              trigger="hover"
+              disabled={!isInDistributionPeriod}
+            >
```

Comment:
Shouldn't we hide the popover button completely when we're not in distribution period?

Reply:
What do you mean? With the disabled option, the popover is not shown at all.